src/com/franz/sud/components/engine/GameInitializer.java

```diff
@@ -433,23 +431,22 @@ public void setupItems() {
         items.put("talisman", talisman);
         items.put("talismanEvasion", talismanEvasion);
 
-        gameInventory.addItemToGameInventory(commonSword);
-        gameInventory.addItemToGameInventory(rareSword);
-        gameInventory.addItemToGameInventory(rapier);
-        gameInventory.addItemToGameInventory(chainMail);
-        gameInventory.addItemToGameInventory(breastPlate);
-        gameInventory.addItemToGameInventory(kevlar);
-        gameInventory.addItemToGameInventory(redMoon);
-        gameInventory.addItemToGameInventory(talisman);
-        gameInventory.addItemToGameInventory(talismanEvasion);
-        gameInventory.setHero(hero);
+        inventoryService.addItemToGameInventory(commonSword);
+        inventoryService.addItemToGameInventory(rareSword);
+        inventoryService.addItemToGameInventory(rapier);
+        inventoryService.addItemToGameInventory(chainMail);
+        inventoryService.addItemToGameInventory(breastPlate);
+        inventoryService.addItemToGameInventory(kevlar);
+        inventoryService.addItemToGameInventory(redMoon);
+        inventoryService.addItemToGameInventory(talisman);
+        inventoryService.addItemToGameInventory(talismanEvasion);
+        inventoryService.setHero(hero);
     }
 
     /**
      * Sets up the game dialogue for each progress.
      */
-    @Override
-    public void setupDialogue() {
+    private void setupDialogue() {
 
         // Sets the String array of each progress/room.
         String[] nullNarrative = new String[] {};

```

Comment:
What's the purpose of `nullNarrative`?

I can't comment in the appropriate lines so I'll put it here:
* There's a typo in `foundAnItem`, `enemySlain`, etc.
* ```
   String[] enemyFound = new String[] {
                "Watch out! There's a banshee waiting for you at the corner",
   ```
   Is there a chance it will show this even if it's a different enemy?